src/components/SpectralProfiler/SpectralProfilerSettingsPanelComponent/SpectralProfilerSettingsPanelComponent.tsx

```diff
@@ -79,6 +82,70 @@ export class SpectralProfilerSettingsPanelComponent extends React.Component<Widg
         this.widgetStore.setUseWcsValues(changeEvent.target.checked);
     };
 
+    handleXMinChange = (ev) => {
```

Comment:
could we add explicit types to these `ev` values? It might make maintenance of the functions easier